components/prism-bsl.js

```diff
@@ -0,0 +1,71 @@
+// 1C:Enterprise
+// https://github.com/Diversus23/
+//
+Prism.languages.bsl = {
+	'comment': /\/\/.*/,
+	'string': [
+		// Строки
+		// Strings
+		{
+			pattern: /"(?:[^"]|"")*"(?!")/,
+			greedy: true
+		},
+		// Дата и время
+		// Date & time
+		{
+			pattern: /'(?:[^'\r\n\\]|\\.)*'/
+		}
+	],
+	'keyword': [
+		{
+			// RU
+			pattern: /(^|[^\w\u0400-\u0484\u0487-\u052f\u1c80-\u1c88\u1d2b\u1d78\u2de0-\u2dff\ua640-\ua69f\ufe2e\ufe2f])(?:пока|для|новый|прервать|попытка|исключение|вызватьисключение|иначе|конецпопытки|неопределено|функция|перем|возврат|конецфункции|если|иначеесли|процедура|конецпроцедуры|тогда|знач|экспорт|конецесли|из|каждого|истина|ложь|по|цикл|конеццикла|выполнить)(?![\w\u0400-\u0484\u0487-\u052f\u1c80-\u1c88\u1d2b\u1d78\u2de0-\u2dff\ua640-\ua69f\ufe2e\ufe2f])/i,
+			lookbehind: true
+		},
+		{
+			// EN
+			pattern: /\b(?:while|for|new|break|try|except|raise|else|endtry|undefined|function|var|return|endfunction|null|if|elseif|procedure|endprocedure|then|val|export|endif|in|each|true|false|to|do|enddo|execute)\b/i
```

Comment:
As I pointed out [here](https://github.com/PrismJS/prism/pull/2520#discussion_r471029326), you might want to replace `\b` or is there a reason against this?

Reply:
I wanted to explicitly separate EN and RU constructs. I think this will be correct and easier to follow in the future.
And since \b is used everywhere, for EN, I do the same.